gluoncv/model_zoo/mobilenetv3.py

```diff
@@ -0,0 +1,307 @@
+            return out
+
+    def _get_pad(self, kernel_size):
+        if kernel_size == 1:
+            return 0
+        elif kernel_size == 3:
+            return 1
+        elif kernel_size == 5:
+            return 2
+        elif kernel_size == 7:
+            return 3
+
+
+class _MobileNetV3(HybridBlock):
+    def __init__(self, cfg, cls_ch_squeeze, cls_ch_expand, multiplier=1.,
+                 classes=1000,
+                 norm_kwargs=None, last_gamma=False,
+                 final_drop=0., use_global_stats=False,
+                 name_prefix='', **kwargs):
+        super(_MobileNetV3, self).__init__(prefix=name_prefix)
+        norm_kwargs = norm_kwargs if norm_kwargs is not None else {}
+        if use_global_stats:
+            norm_kwargs['use_global_stats'] = True
+        # initialize residual networks
+        k = multiplier
+        self.last_gamma = last_gamma
+        self.norm_kwargs = norm_kwargs
+        self.inplanes = 16
+
+        with self.name_scope():
+            self.conv = nn.HybridSequential()
+            self.conv.add(nn.Conv2D(channels=make_divisible(k*self.inplanes), \
+                          kernel_size=3, padding=1, strides=2,
+                                    use_bias=False, prefix='first-3x3-conv-conv2d_'))
+            self.conv.add(nn.BatchNorm(prefix='first-3x3-conv-batchnorm_'))
+            self.conv.add(HardSwish())
+            i = 0
+            for layer_cfg in cfg:
+                layer = self._make_layer(kernel_size=layer_cfg[0],
+                                         exp_ch=make_divisible(k * layer_cfg[1]),
+                                         out_channel=make_divisible(k * layer_cfg[2]),
+                                         use_se=layer_cfg[3],
+                                         act_func=layer_cfg[4],
+                                         stride=layer_cfg[5],
+                                         prefix='seq-%d'%i,
+                                        )
+                self.conv.add(layer)
+                i += 1
+            self.cls = nn.HybridSequential()
+            self.cls.add(nn.Conv2D(channels= \
+                         make_divisible(k*cls_ch_squeeze), \
+                         kernel_size=1, padding=0, strides=1,
+                                   use_bias=False, prefix='last-1x1-conv1-conv2d_'))
+            self.cls.add(nn.BatchNorm(prefix='last-1x1-conv1-batchnorm_',
+                                      **({} if norm_kwargs is None else norm_kwargs)))
+            self.cls.add(HardSwish())
+            self.cls.add(nn.GlobalAvgPool2D())
+            self.cls.add(nn.Conv2D(channels=cls_ch_expand, kernel_size=1, padding=0, strides=1,
+                                   use_bias=False, prefix='last-1x1-conv2-conv2d_'))
+            self.cls.add(HardSwish())
+
+            if final_drop > 0:
+                self.cls.add(nn.Dropout(final_drop))
+            self.cls.add(nn.Conv2D(in_channels=cls_ch_expand, channels=classes,
```

Comment:
it's better to

1. have `self.features` instead of `self.cls`
2. move the last `Conv2D` out of `self.features`, and with `Flatten` merged as `self.output`